samples/RaspberryPiInstructions.md

```diff
@@ -26,6 +27,8 @@ These steps have been tested on a RPi 2 and RPi 3 with Linux and Windows.
 </Project>
 ```
 
+* Add a new package source to the project with source: `https://dotnet.myget.org/F/dotnet-core/api/v3/index.json`.
```

Comment:
I'm actually thinking it may be best to give an example of a complete but minumal NuGet.config here telling people that they should have a file by that name next to their csproj. Can you make that update?